simplq/src/components/pages/Status/QueueStatus.jsx

```diff
@@ -98,9 +98,8 @@ function QueueStatus(props) {
         aheadCount={tokenStatusResponse.aheadCount}
       />
       {renderButtons()}
-      <QueueDetails />
+      <QueueDetails queueName={tokenStatusResponse.queueName} />
```

Comment:
let's pass down the entire tokenStatusResponse object... it's of the following structure: 

```
{
    "tokenId": "24895b64-5242-4834-8295-b60f8e9f7037",
    "tokenNumber": 6,
    "tokenStatus": "REMOVED",
    "queueName": "NavaQ",
    "aheadCount": null,
    "notifiable": false
}
```

Reply:
Completely ignore the above comment, I've left another comment in the QueueDetails component file.